web/modules/custom/dpl_event/src/PriceFormatter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\dpl_event;
+
+use Brick\Math\BigDecimal;
+use Drupal\Core\StringTranslation\TranslationInterface;
+
+/**
+ * Formats prices according to local rules.
+ *
+ * This is important as we do not rely on fields built for prices. Instead we
+ * have double fields with values as strings.
+ */
+class PriceFormatter {
+
+  /**
+   * Constructor.
+   */
+  public function __construct(
+    protected TranslationInterface $translation,
+  ) {}
+
+  /**
+   * Format a single price.
+   */
+  public function formatPrice(string $price_string): string {
+    $context = ['context' => 'dpl_event'];
```

Comment:
either rename it `$translation_context`, or move the array into the `translate()` as a parameter

Reply:
I opted for `$translation_options` as that is what the variable is used for. It contains the context.